### PR TITLE
prevent networkmanager from controlling eth0

### PR DESCRIPTION
**Pull request by jeffreyzhou**
Update device eth0 interface configuration, prevent networkmanager from controlling eth0

### CODE REVIEW

**Comment by nirajdp76** [comment]
Add description to PR stating problem trying to address.

File: roles/common/tasks/networking.yml [hunk 1]
```diff
@@ -126,6 +126,20 @@
     when: ansible_default_ipv4.network  != ( item.cidr | ipaddr('network'))
   when: ursula_os == 'rhel' and (update_routes|default('false')|bool)
 
+- block:
```
Comment (nirajdp76) [inline roles/common/tasks/networking.yml]: Why are we doing this here? This should be in our CI predeploy hook.

Comment (jeffreyzhou) [inline roles/common/tasks/networking.yml]: Move to ci-full-rhel pre-deploy.

**Comment by nirajdp76** [comment]
don;t we need this for other rhel envs? (ceph and ceph_swift)?

**Comment by jeffreyzhou** [comment]
added for ci-ceph-redhat and ceph-swift-rhel as well

**Pull request merged**